examples/neural_compressor/question-answering/README.md

```diff
@@ -55,6 +55,8 @@ python run_qa.py \
     --output_dir /tmp/squad_output
 ```
 
+The flag `--backend` can be passed along and set to `ipex` to use INC IPEX backend. Default backend is `pytorch_fx` if value not specified.
```

Comment:
Why not use a flag `--use_ipex` to make it clear that it adds the capacity to use the ipex backend ? Also should add more information about the gains ?

Reply:
We now use the flag `--use_ipex`. We set `config.use_ipex` to `False` in `train()` and `evaluate()` in `trainer.py` to avoid bf16 issues when calling `trainer.train()` and `trainer.evaluate()`.